modules/nw-sriov-nic-mlx-secure-boot.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+// :_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-nic-mlx-secure-boot_{context}"]
+= Configuring the SR-IOV Network Operator on Mellanox cards when Secure Boot is enabled
+
+The SR-IOV Network Operator supports an option to skip the firmware configuration for Mellanox (MLX) devices. It is currently the only way to create virtual functions by using the SR-IOV Network operator, if the system has secure boot enabled. You are required to manually configure and allocate the number of virtual functions in the firmware before switching the system to secure boot.
+
+[NOTE]
+====
+The number of virtual functions in the firmware will be the maximum number of virtual functions that you can request in the policy.
+====
+
+.Procedure
+
+. Configure the virtual functions (VFs).
+
+.. Run the following command when the system is without a secure boot when using the sriov-config daemon:
++
+[source,terminal]
+----
+mstconfig -d -0001:b1:00.0 set SRIOV_EN=1 NUM_OF_VFS=16
+----
+
+. Configure the SR-IOV Network Operator with the Mellanox plugin disabled.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+spec:
+  configDaemonNodeSelector: {}
+  configurationMode: daemon
+  disableDrain: false
+  disablePlugins:
+  - mellanox
+  enableInjector: true
+  enableOperatorWebhook: true
+  logLevel: 2
+----
+
+. Reboot the system to enable the virtual functions and the configuration settings.
+
+. Check the virtual functions (VFs) after rebooting.
++
+[source,terminal]
+----
+# oc -n openshift-sriov-network-operator get sriovnetworknodestate.sriovnetwork.openshift.io  cnfdr14.telco5g.eng.rdu2.redhat.com -oyaml
+----
++
+[source,yaml]
+----
+- deviceID: 101d
+    driver: mlx5_core
+    eSwitchMode: legacy
+    linkSpeed: -1 Mb/s
+    linkType: ETH
+    mac: 08:c0:eb:96:31:25
+    mtu: 1500
+    name: ens3f1np1
+    pciAddress: 0000:b1:00.1
+    totalvfs: 16
```

Comment:
please add a note to point this is the number that was configured in the mstconfig for the firmware

Reply:
@ogradyp not the `pciAddress` it's the `totalvfs` it should be the same number that the customer apply in `NUM_OF_VFS=16` 

command from above

`mstconfig -d -0001:b1:00.1 set SRIOV_EN=1 NUM_OF_VFS=16 <1> <2>`

Reply:
Thanks @SchSeba for your feedback. I have applied your suggestion.